vendor.conf

```diff
@@ -34,19 +34,19 @@ github.com/google/uuid 0cd6bf5da1e1c83f8b45653022c74f71af0538a4 # v1.1.1
 github.com/golang/protobuf aa810b61a9c79d51363740d207bb46cf8e620ed5 # v1.2.0
 github.com/gogo/protobuf ba06b47c162d49f2af050fb4c75bcbc86a159d5c # v1.2.1
 github.com/gogo/googleapis d31c731455cb061f42baff3bda55bad0118b126b # v1.2.0
-github.com/godbus/dbus c7fdd8b5cd55e87b4e1f4e372cdb1db61dd6c66f
+github.com/godbus/dbus/v5 37bf87eef99d69c4f1d3528bd66e3a87dc201472 # v5.0.3
 github.com/docker/go-units 519db1ee28dcc9fd2474ae59fca29a810482bfb1 # v0.4.0
 github.com/docker/go-metrics 4ea375f7759c82740c893fc030bc37088d2ec098
 github.com/docker/go-events 9461782956ad83b30282bf90e31fa6a70c255ba9
-github.com/coreos/go-systemd 48702e0da86bd25e76cfef347e2adeb434a0d0a6
+github.com/coreos/go-systemd/v22 2d78030078ef61b3cae27f42ad6d0e46db51b339 # v22.0.0
 github.com/containerd/typeurl a93fcdb778cd272c6e9b3028b2f42d813e785d40
 github.com/containerd/ttrpc 92c8520ef9f86600c650dd540266a007bf03670f
 github.com/containerd/go-runc a5c2862aed5e6358b305b0e16bfce58e0549b1cd
 github.com/containerd/fifo bda0ff6ed73c67bfb5e62bc9c697f146b7fd7f13
 github.com/containerd/continuity f2a389ac0a02ce21c09edd7344677a601970f41c
-github.com/containerd/containerd 0a1f2b40642e54ed06cd0a22cdf6025cbe70853c
+github.com/containerd/containerd e1221e69a824ce9aaca34c5bb603feb2f921b883
 github.com/containerd/console 8375c3424e4d7b114e8a90a4a40c8e1b40d1d4e6
-github.com/containerd/cgroups fada802a7909d430bd17126fd6e4bbf5716f2bcd
+github.com/containerd/cgroups 7347743e5d1e8500d9f27c8e748e689ed991d92b
 github.com/beorn7/perks 4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9
 github.com/Microsoft/hcsshim d2849cbdb9dfe5f513292a9610ca2eb734cdd1e7
```

Comment:
to move up containerd vndr to e122 we also need:
https://github.com/containerd/containerd/commit/4d62d8795c42dfec2506631cc7013b3b8786bf76
+github.com/Microsoft/hcsshim                        b3f49c06ffaeef24d09c6c08ec8ec8425a0303e2 # v0.8.7

Reply:
@mikebrow - I've never know how to do this myself. Is the goal here that we track the vendor versions that containerd defines and update to them at the same cadence as containerd? So when we make a containerd vendor we should also take any vendor updates that the containerd vendor.conf has as well?

Reply:
basically when you want to move up vendor.. (for now, should go away if we ever move / merge containerd/cri) you have to take a look at commit changes to containerd's vendor.conf.  Normally when we/I move up we move up to the latest commit, bring up vendor.conf.. hit blame.. look at the recent version changes and compare with our vendor.conf. At one point we had a script that would do that automatically, it's broken and I haven't looked at why it's broke yet :-)  Manual checking isn't to bad.  

In this case he was just moving up a couple commits for containerd.. so I opened up the containerd commits checking for vendor changes and found that ^ one.

Reply:
No prob. I will do this from now on too! Thanks

Reply:
done